bentoml/saved_bundle/bundler.py

```diff
@@ -61,6 +67,18 @@ def save_to_dir(bento_service, path, version=None, silent=False):
         raise BentoMLException(
             "save_to_dir only works with instances of custom BentoService class"
         )
+    remote_save_path = None
```

Comment:
can you refactor this to make it cleaner?  I think we should be able to avoid this